dev/tab1tha/main.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import sklearn
+import seaborn as sns
+import k_nn
+from sklearn.model_selection import train_test_split
+
+#import data into a pandas dataframe
+df = pd.read_csv('..\..\datasets\generated.csv')
```

Comment:
nit: Please break out the local filepath into a variable and then pass that variable. This will make local testing easier for reviewers.

Reply:
This has been solved using the pathlib builtin module